### PR TITLE
Simplified mobile wallet

### PR DESCRIPTION
**Pull request by arcbtc**
Will make advanced stuff optional, but cleaned UI for mobile

### CODE REVIEW

**Comment by arcbtc** [comment]
Uploading simplemobile.mp4…

**Comment by prusnak** [comment]
> https://github.com/lnbits/lnbits/assets/33088785/43012ea3-0e47-4682-bd13-b074d685679e

Love it!

File: lnbits/core/templates/core/wallet.html [hunk 1]
```diff
@@ -62,6 +71,15 @@ <h3 class="q-my-none">
                 style="opacity: 0.75"
               ></span>
             </div>
+            <q-btn
+              @Click="mobileSimple = !mobileSimple"
```
Comment (motorina0) [inline lnbits/core/templates/core/wallet.html:75]:
```suggestion
              @click="mobileSimple = !mobileSimple"
```

File: lnbits/core/templates/core/wallet.html [hunk 2]
```diff
@@ -62,6 +71,15 @@ <h3 class="q-my-none">
...
+              class="float-right lt-md"
+              size="sm"
+              flat
+              style="color: primary"
```
Comment (motorina0) [inline lnbits/core/templates/core/wallet.html:79]: in other places this is used:
```css
color="primary"
```

**Comment by motorina0** [comment]
Suggestion:
 - align center `amount` and `Less` button also when the user is on the `More` mode (looks fine in the compact mode)
 
image

**Comment by motorina0** [comment]
Suggestion 2:
 - add an `Show/Hide API` toggle. The API panel is important but rarely used, it should not take so much screen space.
 
image

**Comment by motorina0** [comment]
Suggestion 3:
 - nitpick: transition animation between More/Less (no idea how the CSS would look like)

**Comment by motorina0** [comment]
Suggestion 4:
 - this is likely the second most important screen (on mobile)
 - maybe some fields should only be shown on a More/Less basis

image

**Comment by arcbtc** [comment]
Added last transaction as well

https://github.com/lnbits/lnbits/assets/33088785/26f15cb5-8398-46cb-9b4c-382328fa6bc3

**Comment by arcbtc** [comment]
> Suggestion 2:
> 
>     * add an `Show/Hide API` toggle. The API panel is important but rarely used, it should not take so much screen space.
> 
> 
> image

Thought about it as well, but I kinda like having this stuff, its where you set the fiat currency and can scan the wallet link. People on mobile need to be able to access.

**Comment by motorina0** [comment]
Maybe the wallet name (of the current wallet) should be visible somewhere (on mobile).

**Comment by arcbtc** [comment]
> Maybe the wallet name (of the current wallet) should be visible somewhere (on mobile).

Its in "more"

File: lnbits/core/templates/core/wallet.html (hunk 1 above)
Comment (prusnak) [reply lnbits/core/templates/core/wallet.html]: addressed in 3de064d6ca90d6c378912ede841f89b471445e6a

File: lnbits/core/templates/core/wallet.html (hunk 2 above)
Comment (prusnak) [reply lnbits/core/templates/core/wallet.html]: addressed in a47860df3f36a40862a1b85c33525e359b79f4e1

**Review by motorina0 — APPROVED** [review]

**Review by talvasconcelos — APPROVED** [review]
hell yeah!

**Pull request merged**